src/software/thunderscope/play/playinfo_widget.py

```diff
@@ -0,0 +1,84 @@
+import pyqtgraph as pg
+import pyqtgraph.console as pg_console
+from proto.play_info_msg_pb2 import PlayInfo
+from software.networking.threaded_unix_listener import ThreadedUnixListener
+import software.thunderscope.constants as constants
+from google.protobuf.json_format import MessageToDict
+from pyqtgraph.Qt import QtCore, QtGui
+from pyqtgraph.Qt.QtWidgets import *
+import queue
+
+from proto.robot_log_msg_pb2 import RobotLog
+
+
+class playInfoWidget(QTableWidget):
+
+    NUM_ROWS = 6
```

Comment:
Ideally the number of rows should be dynamically set based on the number of robots, so in the future the widget can support more robots. Though this is not a high priority right now, so I have created #2560 so it can be worked on in another PR. Could you add this as a todo to the code?

Reply:
Thanks, added!

Reply:
Could you add the ticket number to the TODO aswell?